tests/model_hub_tests/models_hub_common/test_performance_model.py

```diff
@@ -0,0 +1,200 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+import gc
+import os
+import sys
+import time
+from enum import Enum
+import traceback
+import pytest
+
+import numpy as np
+from models_hub_common.multiprocessing_utils import multiprocessing_run
+import openvino as ov
+
+import tensorflow_text  # do not delete, needed for text models
```

Comment:
```suggestion
# noinspection PyUnresolvedReferences
import tensorflow_text  # do not delete, needed for text models
```

Reply:
please add `# noinspection PyUnresolvedReferences` so it will not be deleted durng PyCharm formatting